src/components/Slider.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled, {use} from 'reshadow';
+
+import {checkBoxStyles} from '#/components/Trigger';
+
+type OptionProps = ({
+  active: boolean,
+  label: string,
+  onClick: (option: string) => void,
+});
+
+const Option: React.FC<OptionProps> = ({
+  active = false,
+  label,
+  onClick,
+}: OptionProps) => styled(
+  checkBoxStyles.clickablePannel
+)`
+  |pane[active] {
+    cursor: default;
+  }
+`(
+  <use.pane
+    active={active ? 'true' : undefined}
+    onClick={() => { onClick(label); }}
+  >
+    {label}
+  </use.pane>
+);
+
+type SliderProps = {
+  options: readonly string[],
+  chosenOption: string,
+  chooseOption: (option: string) => void,
```

Comment:
Bad naming: use synonym for `chosenOption` or `chooseOption`

Reply:
Fixed here: 38a0fb5e35cc2e31425bd81d6a1dcc36ccef6345.